internal/test/testdata/index/incomplete_files.yaml

```diff
@@ -1,9 +1,9 @@
 ---
 wantErrList:
   loadFailures:
-    - error: "invalid Policy.DerivedRoles: embedded message failed validation | caused by: invalid DerivedRoles.Definitions: value must contain at least 1 item(s)"
+    - error: "jsonschema: '/derivedRoles' does not validate with https://api.cerbos.dev/cerbos/policy/v1/Policy.schema.json#/allOf/0/properties/derivedRoles/$ref/required: missing properties: 'definitions'"
```

Comment:
We should clean up this error message like we do in https://github.com/cerbos/cerbos/blob/b6fdc81fb3ab0245ff920107ea9bc4b8afe20c5f/internal/schema/error.go#L43. The user doesn't care which schema was used because they can't control it. Just need to tell them the actual problem.